lib/shared/bucketing-assembly-script/project.json

```diff
@@ -30,10 +30,10 @@
             "options": {
                 "commands": [
                     {
-                      "command": "asc assembly/index.ts --target debug --exportRuntime --initialMemory 100 --maximumMemory 10000 --bindings esm"
+                      "command": "asc assembly/index.ts --target debug --exportRuntime --initialMemory 10000 --maximumMemory 20000 --bindings esm"
```

Comment:
not 50k max?

Reply:
It seems to work either way, do we need a higher limit? Was that based on the math you did?

Reply:
50k was based on the GH usage math - more futureproofing than anything

Reply:
50K pages would be 3.2 GB of memory... (each page is 64kb)

Reply:
Yep - this is assuming that each config is 2-3MB - or about 45 pages. Basing it off GH/RBC's per second request load - we can assume about 1200 users would need to be bucketed at any one time - 45 * 1200 = ~50k pages of RAM

Reply:
This is the scale issue i was talking about

Reply:
but they will all be using the same cached config.

Reply:
but also -50k isn't automatically allocated - it's just the max that the GC can allocate total at any one time